### PR TITLE
Author public development article

### PR DESCRIPTION
**Pull request by pholleran**
This PR partially addresses #217, specifically focusing on public development. I have focused primarily on the human communication concerns around public development. A brief discussion of the security concerns links off to other resources within, and outside of FINOS.

One or more future PRs will address other concerns highlighted in #217, including:

* outbound security
* good practices when contributing a new project

Looking forward to the community's suggestions and feedback.

### CODE REVIEW

**Review by psmulovics — APPROVED** [review]

**Comment by robmoffat** [comment]
Thanks for this, @pholleran .  @psmulovics can we add to the agenda for next Wednesday and discuss?

File: docs/bok/Activities/Level-3/Public-Development.md [hunk 1]
```diff
@@ -8,32 +8,59 @@ sidebar_position: 1
...
+
+Personal risk is related to any fear or concern an individual may have around participation in an open source community. Putting your work for the world can, at first, be a bit unnerving for some. Common concerns include not knowing what to do or doing the "wrong thing," receiving less than constructive feedback, or having their contribution rejected. 
+
+Healthy open source communities actively work to promote positive environments for contributors by:
+
+* Helping everyone understand the project's norms, standards, and processes
+* Setting and enforcing norms for communication
+* Pointing new contributors to small, easier to tackle opportunities
+
+GitHub has published a helpful blog of [tips for creating a healthy and sustainable open source community](https://github.blog/2022-07-07-healthy-and-sustainable-communities/). It is helpful for new contributors and maintainers alike to familiarize themselves with these practices and embody them in their open source work.
```
Comment (pholleran) [inline docs/bok/Activities/Level-3/Public-Development.md:45]: Helpful is used in two sentences in a row. Should modify one of the two sentences to reduce duplication.

File: docs/bok/Activities/Level-3/Public-Development.md [hunk 2]
```diff
@@ -8,32 +8,59 @@ sidebar_position: 1
...
+### Working with others
 
-something about using GitHub or GitLab.
+The biggest thing to remember about open source development is that it happens in the public sphere, in written text, with other humans. Text can be a challenging medium in which to interact with others. Tone and intent can be hard to communicate - emoji help. Language barriers often exist. For this reason it is important open source developers strive for clarity, compassion, and empathy in their communication.
 
-Risk of DLP. (we have articles already on this, we need to reference)
+Many folks have written in depth on this topic. The following articles dig a bit deeper into the human side of open, collaborative development:
 
-small PRs.
+* [Why the success of open source depends on empathy](https://opensource.com/article/21/2/open-source-empathy), by Bronagh Sorota
```
Comment (mimiflynn) [inline docs/bok/Activities/Level-3/Public-Development.md:20]: Relevant Linux Foundation training regarding OSS:

- [Inclusive Open Source Community Orientation (LFC102)](https://trainingportal.linuxfoundation.org/courses/inclusive-open-source-community-orientation-lfc102)
- [Inclusive Strategies for Open Source (LFC103)](https://trainingportal.linuxfoundation.org/courses/inclusive-strategies-for-open-source-lfc103)

**Comment by katnovakovic** [comment]
Nice article! Some feedback:

- **Clearer title**: From the title, I wasn't sure what the article was about and it went in a different direction than I expected. Perhaps something like "Public Development Communications"? 

- **Firm Policies**: Perhaps add something about referring to your firm's internal policies around communications and social media, to protect against brand and personal reputation as well as other risks already mentioned.

**Comment by robmoffat** [comment]
@pholleran any chance you can push this one over the line?  Would be nice to close it out, thanks

File: docs/bok/Activities/Level-3/Public-Development.md (hunk 1 above)
Comment (pholleran) [reply docs/bok/Activities/Level-3/Public-Development.md]: Included, thank you 🙇‍♂️

**Comment by pholleran** [comment]
@robmoffat thanks for the push. Ready for one more review.

**Review by robmoffat — APPROVED** [review]

**Pull request merged**